js/mainview.js

```diff
@@ -0,0 +1,24 @@
+$('.frame').mousedown(function(){
```

Comment:
Maintain spacing before bracket (not just here).

Reply:
fixed.